sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample4_TrainModel.md

```diff
@@ -0,0 +1,98 @@
+
+```C# Snippet:FormRecognizerSample4TrainModelWithForms
+// For instructions on setting up forms for training in an Azure Storage Blob Container, see
+// https://docs.microsoft.com/azure/cognitive-services/form-recognizer/quickstarts/curl-train-extract#train-a-form-recognizer-model
+
+FormTrainingClient client = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+CustomFormModel model = await client.StartTrainingAsync(new Uri(trainingFileUrl)).WaitForCompletionAsync();
+
+Console.WriteLine($"Custom Model Info:");
+Console.WriteLine($"    Model Id: {model.ModelId}");
+Console.WriteLine($"    Model Status: {model.Status}");
+Console.WriteLine($"    Created On: {model.CreatedOn}");
+Console.WriteLine($"    Last Modified: {model.LastModified}");
+
+foreach (CustomFormSubModel subModel in model.Models)
+{
+    Console.WriteLine($"SubModel Form Type: {subModel.FormType}");
+    foreach (CustomFormModelField field in subModel.Fields.Values)
+    {
+        Console.Write($"    FieldName: {field.Name}");
+        if (field.Label != null)
+        {
+            Console.Write($", FieldLabel: {field.Label}");
+        }
+        Console.WriteLine("");
+    }
+}
+```
+
+## Train a model with forms and labels
+
+Train custom models to recognize specific fields and values you specify by labeling your custom forms. A `CustomFormModel` is returned indicating the fields the model will extract, as well as the estimated accuracy for each field.
+
+```C# Snippet:FormRecognizerSample5TrainModelWithFormsAndLabels
+// For instructions to set up forms for training in an Azure Storage Blob Container, please see:
+// https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/curl-train-extract#train-a-form-recognizer-model
+
+// For instructions to create a label file for your training forms, please see:
+// https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/label-tool
+
+FormTrainingClient client = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+CustomFormModel model = await client.StartTrainingAsync(new Uri(trainingFileUrl), useLabels: true).WaitForCompletionAsync();
+
+Console.WriteLine($"Custom Model Info:");
+Console.WriteLine($"    Model Id: {model.ModelId}");
+Console.WriteLine($"    Model Status: {model.Status}");
+Console.WriteLine($"    Created On: {model.CreatedOn}");
+Console.WriteLine($"    Last Modified: {model.LastModified}");
+
+foreach (CustomFormSubModel subModel in model.Models)
+{
+    Console.WriteLine($"SubModel Form Type: {subModel.FormType}");
+    foreach (CustomFormModelField field in subModel.Fields.Values)
+    {
+        Console.Write($"    FieldName: {field.Name}");
+        if (field.Accuracy != null)
+        {
+            Console.Write($", Accuracy: {field.Accuracy}");
+        }
+        Console.WriteLine("");
+    }
+}
+
+// Delete the model on completion.
```

Comment:
It looks like we're deleting the model at the end in this snippet but not the "with forms" snippet.  I don't have a strong preference which way it goes in the .md file -- maybe not deleting it since the main use case is to leave it up to use for analysis later on -- but let's be consistent.

Reply:
fixed